onnxruntime/test/contrib_ops/decoder_masked_multihead_attention_op_test.cc

```diff
@@ -640,14 +640,17 @@ TEST(DecoderMaskedSelfAttentionTest, Test_fp32) {
     return;
   }
 
+  std::vector<int> batch_sizes = {1, 4};
+  std::vector<int> past_sequence_lengths = {1, 3000};
+  std::vector<int> hidden_sizes = {768};
```

Comment:
Any reason for trying fewer hidden sizes for fp32 ?

Reply:
I think one hidden size for test should be enough? since it's unlikely to introduce bugs that affect specific hidden size

Reply:
added more cases